apps/server/src/modules/workspaces/controller.ts

```diff
@@ -33,6 +35,11 @@ export class WorkspacesController {
     @Param('name') name: string,
     @Res() res: Response
   ) {
+    this.metric.restRequest(1, {
+      method: 'GET',
+      path: '/api/workspace/:id/blobs/:name',
+      job: 'get workspace blob',
```

Comment:
we know what each controller does, the job field seems useless

Reply:
Updated, haven't notice that!